cypress/support/helpers/runTestsForPage.js

```diff
@@ -1,11 +1,19 @@
-import config from './config/services';
+import config from '../config/services';
 import shouldSmokeTest from './shouldSmokeTest';
-import { runCommonTests, runCommonCanonicalTests } from '../integration/pages';
+import {
+  runCommonTests,
+  runCommonCanonicalTests,
+} from '../../integration/pages';
 
 const serviceHasPageType = (service, pageType) =>
   config[service].pageTypes[pageType].path !== undefined;
 
-const iterator = (pageType, runTests, runCanonicalTests, runAmpTests) => {
+const runTestsForPage = (
+  pageType,
```

Comment:
Since you're renaming things, I'd suggest this as clearer naming. Note the syntax change so that it's clear what is being passed in, when this function is called. 
```
({
  pageType,
  testsForAllPlatforms,
  testsForAmpOnly,
  testsForCanonicalOnly, 
})

`testsForCanonical`